### PR TITLE
Telink b91 I2C Driver Support

### PR DESCRIPTION
**Pull request by yurvyn**
I2C Driver implementation for Telink B91 SoC

### CODE REVIEW

**Comment by yurvyn** [comment]
Please approve the PR if you are OK with the changes.

**Comment by yurvyn** [comment]
Two approvals required.
Could anyone please approve the PR?

**Comment by yurvyn** [comment]
@kgugala, @pgielda
Since you are owners of most of the files in this PR, could you please approve it?
Thanks.

**Comment by yurvyn** [comment]
@galak, @kgugala, @nashif, @pgielda

There no any activity in this PR for 26 days already.
Could you please assign someone for this PR?

**Review by mbolivar-nordic — CHANGES_REQUESTED** [review]
I noticed a few minor things, otherwise looks fine to me.

File: drivers/i2c/i2c_b91.c
Comment (yurvyn) [reply drivers/i2c/i2c_b91.c]: Done.

File: drivers/i2c/i2c_b91.c
Comment (yurvyn) [reply drivers/i2c/i2c_b91.c]: Done.

File: drivers/i2c/i2c_b91.c
Comment (yurvyn) [reply drivers/i2c/i2c_b91.c]: Done.

File: drivers/i2c/i2c_b91.c
Comment (yurvyn) [reply drivers/i2c/i2c_b91.c:31]: Done.

File: drivers/i2c/i2c_b91.c
Comment (yurvyn) [reply drivers/i2c/i2c_b91.c]: Done.

**Review by mbolivar-nordic — APPROVED** [review]

**Comment by yurvyn** [comment]
@carlescufi, @tbursztyka 

One more approval is left.
Could you please approver the PR?

**Comment by yurvyn** [comment]
Hello @cfriedt 

This PR is not assigned for more than 30 days, but already has 1 approval.
Could you please approve and merge it?

**Review by cfriedt — APPROVED** [review]

**Pull request merged**
